ffmpeg/filter.h

```diff
@@ -59,7 +59,9 @@ struct output_ctx {
   int64_t drop_ts;     // preroll audio ts to drop
 
   int64_t gop_time, gop_pts_len, next_kf_pts; // for gop reset
-
+#ifdef USE_LVPDNN_
+  int dummy;
```

Comment:
Can be changed to a more descriptive variable name like `is_dnn_profile` or similar

Reply:
changed to is_dnn_profile.